### PR TITLE
Add /projects/{forge} API endpoint

### PR DESCRIPTION
**Pull request by lbarcziova**
Needed for packit/dashboard#61

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]

File: packit_service/service/api/projects.py [hunk 1]
```diff
@@ -78,6 +78,41 @@ def get(self, forge, namespace, repo_name):
...
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT.value, "Projects list follows")
+    @ns.response(HTTPStatus.OK.value, "OK")
+    def get(self, forge):
+        """List of projects of given forge"""
+
+        result = []
+        first, last = indices()
+
+        projects_list = GitProjectModel.get_forge(first, last, forge)
```
Comment (TomasTomecek) [inline packit_service/service/api/projects.py:93]: what's the expectation for the `forge` argument? `https://github.com` or `github.com`?

I'm wondering what would be the most convenient way for API users to use the endpoint

File: tests_requre/database/test_models.py [hunk 2]
```diff
@@ -599,6 +599,18 @@ def test_get_project(clean_before_and_after, a_copr_build_for_pr):
     assert project.project_url == "https://github.com/the-namespace/the-repo-name"
 
 
+def test_get_forge(clean_before_and_after, multiple_forge_projects):
+    projects = GitProjectModel.get_forge(0, 10, "github.com")
```
Comment (TomasTomecek) [inline tests_requre/database/test_models.py:603]: oki, here's my answer :D 

so will `https://github.com` work as well?

File: packit_service/service/api/projects.py [hunk 3]
```diff
@@ -78,6 +78,41 @@ def get(self, forge, namespace, repo_name):
...
 
 
+@ns.route("/<forge>")
+@ns.param("forge", "Git Forge")
+class ProjectsForge(Resource):
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT.value, "Projects list follows")
+    @ns.response(HTTPStatus.OK.value, "OK")
+    def get(self, forge):
+        """List of projects of given forge"""
```
Comment (TomasTomecek) [inline packit_service/service/api/projects.py:88]: Could you please provide a few examples for the forge argument? So we can have them displayed in the docs: https://prod.packit.dev/api/

File: tests_requre/database/test_models.py (hunk 2 above)
Comment (lbarcziova) [reply tests_requre/database/test_models.py:603]: no, this will not work, as in other similar methods

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**